docs/source/tensorboard.rst

```diff
@@ -0,0 +1,9 @@
+torch.utils.tensorboard
+===================================
+
+.. automodule:: torch.utils.tensorboard
+
+.. autoclass:: SummaryWriter
```

Comment:
@ezyang do we need to update this before landing?

Reply:
No. But personally, I would do doc fixes before landing, since they're not likely to break things.